src/pages/attribute-dictionary.js

```diff
@@ -193,45 +196,45 @@ const EventDefinition = memo(({ event, filteredAttribute }) => {
         }
       `}
     >
-      <Link to={`?event=${event.name}`}>
-        <h2
-          as={Link}
-          css={css`
-            position: sticky;
-            top: var(--global-header-height);
-            background: var(--primary-background-color);
-            padding: 1rem 0;
+      <h2
+        css={css`
+          position: sticky;
+          top: var(--global-header-height);
+          background: var(--primary-background-color);
+          padding: 1rem 0;
 
-            // cover up the right table border
-            margin-right: -1px;
+          // cover up the right table border
+          margin-right: -1px;
 
-            &:hover svg {
-              opacity: 1;
-            }
+          &:hover svg {
+            opacity: 1;
+          }
 
-            @media (max-width: 1240px) {
-              position: static;
-            }
+          @media (max-width: 1240px) {
+            position: relative;
+          }
+        `}
+      >
+        <div
+          css={css`
+            position: relative;
           `}
         >
+          <Link
```

Comment:
this is adding the embedded links for the h2 elements to the event name

Reply:
They were already there, but the design on it was not consistent with other pages:

Here is a [regular doc](https://docs-preview.newrelic.com/docs/agents/ruby-agent/getting-started/introduction-new-relic-ruby):

<img width="559" alt="Screen Shot 2021-01-11 at 5 09 22 PM" src="https://user-images.githubusercontent.com/565661/104256328-dd6c7200-542f-11eb-8186-5de332c208db.png">


And here is the [attribute dictionary](https://docs-preview.newrelic.com/attribute-dictionary/): 

<img width="361" alt="Screen Shot 2021-01-11 at 5 09 31 PM" src="https://user-images.githubusercontent.com/565661/104256344-e9f0ca80-542f-11eb-9f35-73903758909f.png">


Notice how the link icon is on the other side, not vertically aligned, and much bigger. I was simply trying to unify the design since I was touching links anyways.

Reply:
ahhh that makes more sense. I was going to say that I thought we had this before so I was surprised to see it :)